src/Services/privateApiService.js

```diff
@@ -42,4 +42,26 @@ export const getRequest =  async (url, id = null) => {
     }
 }
 
-
+export const patchApi = async( enpoint, id, data  ) => {
```

Comment:
Para mantener la nomenclatura que vienen trabajando el resto de las personas del equipo la función debería llamarse patchRequest

```suggestion
export const patchRequest = async(enpoint, id, data) => {
```